st2client/st2client/commands/pack.py

```diff
@@ -150,10 +150,28 @@ def __init__(self, resource, *args, **kwargs):
         self.parser.add_argument('name',
                                  help='Name of the %s to create.' %
                                  resource.get_display_name().lower())
+        self.parser.add_argument('--description',
+                                 help='Description of the %s.' %
+                                 resource.get_display_name().lower())
+        self.parser.add_argument('--keywords',
+                                 nargs='+',
+                                 help='Keywords describing the %s.' %
+                                 resource.get_display_name().lower())
+        self.parser.add_argument('--version',
+                                 required=True,
+                                 help='Version of the %s.' %
+                                 resource.get_display_name().lower())
+        self.parser.add_argument('--author',
+                                 required=True,
+                                 help='Author of the %s.' %
+                                 resource.get_display_name().lower())
+        self.parser.add_argument('--email',
+                                 help='%s author\'s email.' %
+                                 resource.get_display_name())
 
     @resource.add_auth_token_to_kwargs_from_cli
     def run(self, args, **kwargs):
-        return self.manager.create(args.name, **kwargs)
+        return self.manager.create(args, **kwargs)
```

Comment:
How hard would it be to make an interactive command here, like `adduser`? I.e. `st2 pack create` or `st2 pack create blabla`, then the user just answers questions instead of writing a huge one-liner? I'm fine with not doing it, but it's a nice feature.